src/NSwag.Sample.NET60Minimal/Program.cs

```diff
@@ -2,16 +2,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using System;
 
-var builder = WebApplication.CreateBuilder(args);
-
-builder.Services.AddEndpointsApiExplorer();
-builder.Services.AddOpenApiDocument(settings =>
-{
-    settings.Title = "Minimal API";
```

Comment:
The flag on this doesn't make sense, the new Startup-less program.cs is used for both controller-based and minimal API code templates.